### DISCUSSION TITLE
Issues about sharing chart

### DISCUSSION THREAD

**Discussion by guoguoguilai** [apache/superset#19178] [Ideas]
Sharing charts with iframe is awesome. I want to make a static site like [this](https://about.gitlab.com/handbook/engineering/development/performance-indicators/#review-to-merge-time-rtmt), and we take Kramdown as the format that supports iframe grammar. And with the update of data, the chart is also updated. 
I found two ways to share charts in Superset. One is share the chart directly, and the data will be updated automaticlly. But it shows all the charts of the dashboard. The other one is "explore chart" with iframe. But in this case, the chart will no longer update since the url is fixed in static site.

How can I make a automaticlly updated chart with a fixed chart url?

**Comment by junlincc** [comment]
To confirm, the request is - to share individual updated chart in iframe from the dashboard?  @guoguoguilai
bringing in our iframe expert @rusackas

**Comment by guoguoguilai** [comment]
> To confirm, the request is - to share individual updated chart in iframe from the dashboard? @guoguoguilai
> bringing in our iframe expert @rusackas

Yes, this is what I want

**Comment by guoguoguilai** [comment]
Any update?
